### PR TITLE
Article screen - Layout only

### PR DESCRIPTION
**Pull request by hernan-clich**
## Summary

Created the complete layout for the `article/:slug` screen, integration is pending.

### Additional comments:

- When a user icon or username is clicked, the page will be redirected to `/user/:username`, I know the actual page currently will take the user to `/:username` but I think this is a change worth considering.
- 

#### When the user is **not** logged in:

3CDF4C23-EAE0-4B15-95C6-4579C034C507_1_105_c

#### When the user _is_ logged in:

pr_article_desk2

#### When the user is **not** logged in:

pr_article_mob2

#### When the user _is_ logged in:

pr_article_mob1

## Trello Card

No card associated with this PR.

### CODE REVIEW

File: src/screens/Article/components/ArticleComments/components/ArticleComment/styles.module.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+.card-footer {
+  background-color: $wild-sand;
+  padding: 12px 20px;
+}
+
+.user-icon {
+  border-radius: 20px;
+  margin-right: 4px;
+  height: 20px;
```
Comment (PipeWolox) [inline src/screens/Article/components/ArticleComments/components/ArticleComment/styles.module.scss:27]: alphabetical order

File: src/screens/Article/components/ArticleContent/components/ArticleBanner/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  font-family: $source-sans-pro;
+  font-size: 45px;
+  font-weight: 600;
+  line-height: 1;
+}
+
+.user-icon {
+  border-radius: 32px;
+  margin-right: 8px;
+  height: 32px;
```
Comment (PipeWolox) [inline src/screens/Article/components/ArticleContent/components/ArticleBanner/styles.module.scss:28]: alphabetical order

**Review by PipeWolox — CHANGES_REQUESTED** [review]
two super simple comments, Excellent work 👌

**Review by PipeWolox — APPROVED** [review]
Great!!

File: src/screens/Article/components/ArticleComments/components/ArticleCommentForm/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
+@import 'scss/fonts/_fonts';
+@import 'scss/variables/_colors';
+
+.container {
+  border: 1px solid $mercury;
+  border-radius: 4px;
+  max-width: 728px;
+  margin: 0 auto 12px;
```
Comment (renzowolox) [inline src/screens/Article/components/ArticleComments/components/ArticleCommentForm/styles.module.scss:8]: Sort

File: src/screens/Article/components/ArticleComments/components/ArticleCommentForm/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+.footer {
+  background-color: $wild-sand;
+  border-top: 1px solid $mercury;
+  padding: 12px 20px;
+}
+
+.user-icon {
+  border-radius: 20px;
+  margin-right: 4px;
+  height: 20px;
```
Comment (renzowolox) [inline src/screens/Article/components/ArticleComments/components/ArticleCommentForm/styles.module.scss:30]: Sort

File: src/screens/Article/components/ArticleComments/components/ArticleCommentForm/styles.module.scss [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  width: 20px;
+}
+
+.button {
+  background-color: $green-fern;
+  border-radius: 3px;
+  color: $white;
+  font-size: 14px;
+  font-weight: 700;
```
Comment (renzowolox) [inline src/screens/Article/components/ArticleComments/components/ArticleCommentForm/styles.module.scss:39]: Sort

File: src/screens/Article/components/ArticleComments/components/ArticlePrompt/styles.module.scss [hunk 6]
```diff
@@ -0,0 +1,29 @@
+@import 'scss/fonts/_fonts';
+@import 'scss/variables/_colors';
+
+.container {
+  max-width: 728px;
+  margin: 0 auto 16px;
```
Comment (renzowolox) [inline src/screens/Article/components/ArticleComments/components/ArticlePrompt/styles.module.scss:6]: Sort

File: src/screens/Article/components/ArticleComments/index.tsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+function ArticleComments() {
+  const isLoggedIn = false;
+  const doesArticleHaveComments = !!MOCKED_COMMENTS && !!MOCKED_COMMENTS.length;
+
+  return (
+    <>
+      {isLoggedIn ? <ArticleCommentForm formData={MOCKED_CURRENT_USER} /> : <ArticlePrompt />}
+      {doesArticleHaveComments
+        ? MOCKED_COMMENTS.map((comment) => <ArticleComment key={comment.id} commentData={comment} />)
+        : null}
```
Comment (renzowolox) [inline src/screens/Article/components/ArticleComments/index.tsx:16]:
```suggestion
      {doesArticleHaveComments
        && MOCKED_COMMENTS.map((comment) => <ArticleComment key={comment.id} commentData={comment} />)}
```

File: src/screens/Article/components/ArticleContent/components/ArticleBanner/styles.module.scss [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+  max-width: $max-content;
+  padding: 15px;
+  width: 95vw;
+}
+
+.title {
+  color: $white;
+  font-family: $source-sans-pro;
+  font-size: 45px;
+  font-weight: 600;
```
Comment (renzowolox) [inline src/screens/Article/components/ArticleContent/components/ArticleBanner/styles.module.scss:21]: Sort

File: src/screens/Article/components/ArticleContent/components/ArticleBanner/styles.module.scss [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+  height: 32px;
+  margin-right: 8px;
+  width: 32px;
+}
+
+.user-link {
+  color: $white;
+  font-family: $source-sans-pro;
+  font-size: 16px;
+  font-weight: 500;
```
Comment (renzowolox) [inline src/screens/Article/components/ArticleContent/components/ArticleBanner/styles.module.scss:36]: Sort

File: src/screens/Article/components/ArticleContent/components/ArticleBanner/styles.module.scss (hunk 8 above)
Comment (hernan-clich) [reply src/screens/Article/components/ArticleContent/components/ArticleBanner/styles.module.scss:21]: This one looks good to me, font-size goes before font-weight.

File: src/screens/Article/components/ArticleComments/components/ArticleCommentForm/styles.module.scss (hunk 3 above)
Comment (hernan-clich) [reply src/screens/Article/components/ArticleComments/components/ArticleCommentForm/styles.module.scss:8]: Re-sorted! 👍

File: src/screens/Article/components/ArticleComments/components/ArticlePrompt/styles.module.scss (hunk 6 above)
Comment (hernan-clich) [reply src/screens/Article/components/ArticleComments/components/ArticlePrompt/styles.module.scss:6]: Re-sorted 👍

**Review by renzowolox — APPROVED** [review]

**Pull request merged**